<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaContatos</class>
 <widget class="QDialog" name="janelaContatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 align=&quot;center&quot; style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:700;&quot;&gt;Dados do cliente:&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_auxiliar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNome">
        <property name="text">
         <string>nome</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sobrenome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSobrenome">
        <property name="text">
         <string>sobrenome</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Estudio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEstudio">
        <property name="text">
         <string>estudio</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Telefone:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTelefone">
        <property name="text">
         <string>telefone</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Estado:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEstado">
        <property name="text">
         <string>estado</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Cidade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCidade">
        <property name="text">
         <string>cidade</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Logradouro:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLogradouro">
        <property name="text">
         <string>logradouro</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Numero:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumero">
        <property name="text">
         <string>numero</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Cep:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCep">
        <property name="text">
         <string>cep</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
